### DISCUSSION TITLE
Welcome to fx-project-Open-Source Discussions!

### DISCUSSION THREAD

**Discussion by fx-project-OS** [fx-project-OS/fx-project-Open-Source#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and optimizations.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rekiahhinton** [comment]
Hi, my name is Rekiah Hinton and I am a new technical writer looking to gain experience. I am looking for an open source project to work on to improve the documentation. Please let me know if you would have anything for me to work on. Thanks
